src/exo/LoopIR_scheduling.py

```diff
@@ -320,6 +320,10 @@ def DoReorderStmt(f_cursor, s_cursor):
     return ir, fwd
 
 
+def DoParallelizeLoop(loop_cursor):
```

Comment:
I think it's fine for now, but we probably need a scheduling operation to turn a parallel loop into a sequential loop